core/src/main/scala/org/apache/spark/broadcast/BroadcastManager.scala

```diff
@@ -52,8 +52,8 @@ private[spark] class BroadcastManager(
 
   private val nextBroadcastId = new AtomicLong(0)
 
-  def newBroadcast[T: ClassTag](value_ : T, isLocal: Boolean): Broadcast[T] = {
-    broadcastFactory.newBroadcast[T](value_, isLocal, nextBroadcastId.getAndIncrement())
+  def newBroadcast[T: ClassTag](_value: T, isLocal: Boolean): Broadcast[T] = {
```

Comment:
I think it's not about adding `_` to the name of private val, but kind of avoiding name conflict if 2 variables have similar meaning. At least this is fine to me, I also do this a lot...